insurance-service/azure-pipelines.yml

```
@@ -0,0 +1,223 @@
+
+- stage: DeployOnTest
+  dependsOn:
+    - Build
+    - BuildImage
+  condition: succeeded()
+  variables:
+    appVersion: $[ stageDependencies.Build.BuildAndTest.outputs['SetVersion.appVersion'] ]
+  jobs:
+  - job: CommitNewTag
+    steps:
+      - checkout: git://gitops
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)